cms/static/js/spec/video/transcripts/file_uploader_spec.js

```diff
@@ -5,7 +5,7 @@ define(
         "xmodule", "jquery.form", "jasmine-jquery"
     ],
 function ($, _, Utils, FileUploader) {
-    describe('Transcripts.FileUploader', function () {
+    xdescribe('Transcripts.FileUploader', function () {
```

Comment:
I've seen before the TODO tagged w/ the JIRA ticket number, and/or the JIRA ticket linked to the PR that adds the TODO. Either approach would be good, in conjunction with adding the TODO.

Note that will mean your build will fail on diff-quality, I'm ok w/ that.